social/apps/django_app/default/fields.py

```diff
@@ -10,8 +14,13 @@
 except ImportError:
     from django.utils.encoding import smart_text
 
+if VERSION >= (1, 8):
+    _JSONFieldBase = models.TextField
+else:  # Django < 1.8, deprecated code, remove it after Django 1.9 release (in December 2015)
```

Comment:
https://www.djangoproject.com/download/

look for supported versions

support for Django < 1.7 is potential security threat for users